docs/developer-guide/plugins.md

```diff
@@ -99,7 +99,7 @@ For your plugin rule to work with the [standard configuration format](../user-gu
 - the primary option
 - optionally, a secondary options object
 
-If your plugin rule supports [autofixing](rules.md#add-autofix), then `ruleFunction` should also accept a third argument: `context`.
+If your plugin rule supports [autofixing](rules.md#add-autofix), then `fixable` should be set to `true` in `meta`.
```

Comment:
[suggest] How about this change for consistency?

https://github.com/stylelint/stylelint/blob/066fc8d8c9a3bf6002b9d13e2f85d9db99cf8f97/docs/developer-guide/rules.md?plain=1#L192

```suggestion
If your plugin rule supports [autofixing](rules.md#add-autofix), then `meta.fixable` should be set to `true`.
```